draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -465,6 +474,35 @@ even when also exporting the expanded key. ML-DSA seed extraction can be
 implemented by including the random seed xi generated at line 1 of Algorithm 1
 ML-DSA.KeyGen in the returned output.
 
+# Pairwise Consistency Testing
+
+This section specifies RECOMMENDED pairwise consistency testing
```

Comment:
I suggest to remove normative "RECOMMENDED". You have a "SHOULD" below which I think is fine.

Reply:
Thanks, removed in [def88ec](https://github.com/lamps-wg/dilithium-certificates/pull/88/commits/def88ec82eab0e26964913430cdc9edbf736eb9b).